src/bo4e/enum/messgroesse.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=missing-module-docstring
+from bo4e.enum.strenum import StrEnum
+
+
+class Messgroesse(StrEnum):
+    """
+    Gibt die physikalische Größe an, die gemessen wurde.
+    """
+
+    STROM = "STROM"  #: STROM
+    SPANNUNG = "SPANNUNG"  #: SPANNUNG
+    WIRKLEISTUNG = "WIRKLEISTUNG"  #: WIRKLEISTUNG
+    BLINDLEISTUNG = "BLINDLEISTUNG"  #: BLINDLEISTUNG
+    DRUCK = "DRUCK"  #: DRUCK
+    LASTGANG = "LASTGANG"  #: LASTGANG
+    LASTPROFIL = "LASTPROFIL"  #: LASTPROFIL
+    TEMPERATUR = "TEMPERATUR"  #: TEMPERATUR
+    ZZAHL = "ZZAHL"  #: Zustandszahl
+    BRENNWERT = "BRENNWERT"  #: BRENNWERT
+    GRADTZAGSZAHLEN = "GRADTZAGSZAHLEN"  #: GRADTZAGSZAHLEN
+    VOLUMENSTROM = "VOLUMENSTROM"  #: VOLUMENSTROM
+    PREISE = "PREISE"  #: PREISE
```

Comment:
```suggestion
    STROM = "STROM"  #: Strom
    SPANNUNG = "SPANNUNG"  #: Spannung
    WIRKLEISTUNG = "WIRKLEISTUNG"  #: Wirkleistung
    BLINDLEISTUNG = "BLINDLEISTUNG"  #: Blindleistung
    DRUCK = "DRUCK"  #: Druck
    LASTGANG = "LASTGANG"  #: Lastgang
    LASTPROFIL = "LASTPROFIL"  #: Lastprofil
    TEMPERATUR = "TEMPERATUR"  #: Temperatur
    ZZAHL = "ZZAHL"  #: Zustandszahl
    BRENNWERT = "BRENNWERT"  #: Brennwert
    GRADTZAGSZAHLEN = "GRADTZAGSZAHLEN"  #: Gradtagszahlen
    VOLUMENSTROM = "VOLUMENSTROM"  #: Volumenstrom
    PREISE = "PREISE"  #: Preise
```

Reply:
Would you like to change the comments to not ALLCAPS ?